examples/plot_bands_functionality.py

```diff
@@ -0,0 +1,107 @@
+"""
+Plot Bands of Satellite Imagery with EarthPy
+==================================================================
+
+Learn how to plot each individual band from satellite imagery with
+EarthPy. This guide will demonstrate how to plot every band from a
+satellite, as well as individual bands. It will also go over some
+of the customization possible when plotting bands.
+
+"""
+
+###############################################################################
+# Plotting Satellite Imagery Bands in Python Using EarthPy
+# ---------------------------------------------------------
+#
+# .. note::
+#    The examples below will show you how to use the ``plot_bands()`` function
+#    to plot individual satellite bands in python. To plot rgb data, read help
+#    documentation related to `ep.plot_rgb()`.
+#
+# The example below walks you through a typical workflow for plotting all bands
+# found in Landsat 8 data with EarthPy.
+#
+# First, you will create a stack of bands using Landsat 8 data and then
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial and plot.
+
+import os
+from glob import glob
+import matplotlib.pyplot as plt
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+# Get data for example
+data = et.data.get_data("vignette-landsat")
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# To get started, make sure your directory is set. Then, create a stack from all of
+# the Landsat .tif files (one per band).
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the Landsat 8 bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_path = glob(
+    "data/vignette-landsat/LC08_L1TP_034032_20160621_20170221_01_T1_sr_band*_crop.tif"
+)
+landsat_path.sort()
+array_stack, meta_data = es.stack(landsat_path)
+
+###############################################################################
+# Plot All Bands in a Stack
+# --------------------------
+#
+# When `ep.plot_bands()` is handed a three dimensional numpy array, as is created
+# when `es.stack()` is ran, it will plot all layers of the numpy array. To title
+# all of the images individually, a list of titles must be submitted as an argument.
+# The list must contain the same number of strings as there are bands in the stack.
+
+titles = ["Ultra Blue", "Blue", "Green", "Red", "NIR", "SWIR 1", "SWIR 2"]
+ep.plot_bands(array_stack, title=titles)
+plt.show()
+
+##################################################################################
+# Plot One Band in a Stack
+# ------------------------
+#
```

Comment:
Use "you" ... active voice

If you give `ep.plot_bands` a one dimensional numpy array, such as one layer
 from the return of ``es.stack``, it will just plot that band. You can turn off the colorbar 
using the cbar parameter (``cbar=False``).